tests/ui/macros/rfc-3086-metavar-expr/syntax-errors.rs

```diff
@@ -125,14 +118,16 @@ macro_rules! open_brackets_without_tokens {
 macro_rules! unknown_count_ident {
     ( $( $i:ident )* ) => {
         ${count(foo)}
-        //~^ ERROR variable `foo` is not recognized in meta-variable expression
+        //~^ ERROR meta-variable expressions must be referenced using a dollar sign
+        //~| ERROR expected expression
     };
 }
 
 macro_rules! unknown_ignore_ident {
     ( $( $i:ident )* ) => {
         ${ignore(bar)}
-        //~^ ERROR variable `bar` is not recognized in meta-variable expression
+        //~^ ERROR meta-variable expressions must be referenced using a dollar sign
+        //~| ERROR expected expression
     };
 }
 
```

Comment:
These tests (which are for "unknown" values) should probably be updated to refer to `$foo` and `$bar` and then additional tests added for the `meta-variable expressions must be referenced using a dollar sign` case.

nit: The error message doesn't seem quite right to me.  The "expression" is `count(_)`, which doesn't need a `$`, but rather it is the *parameter* to the count or ignore method that needs it.  Perhaps `meta-variables within meta-variable expressions must be referenced using a dollar sign`, if that's not too long.

Reply:
Thank you @markbt. The `meta-variable expressions must ...` message has been changed but if possible, I would like to perform further UI modifications in a following PR.